src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -48,10 +48,10 @@
 					v-model="newAcctTerms"
 					:validation="$v.newAcctTerms"
 				>
-					I have read and agree to the Kiva
+					I have read and agree to the {{ !passwordless ? 'Kiva' : '' }}
```

Comment:
This looks like it should be a global change for all users.

Reply:
done